README.md

```diff
@@ -10,53 +16,72 @@ This is part experiment, part solving a personal problem, and part hope that we
 [![Crates.io](https://img.shields.io/crates/v/cargo-binstall.svg)](https://crates.io/crates/cargo-binstall)
 [![Docs.rs](https://docs.rs/cargo-binstall/badge.svg)](https://docs.rs/cargo-binstall)
 
-
-## Getting Started
-
-First you'll need to install `cargo-binstall` either via `cargo install cargo-binstall` (and it'll have to compile, sorry...), or by grabbing a pre-compiled version from the [releases](https://github.com/ryankurte/cargo-binstall/releases) page and putting that somewhere on your path. It's like there's a problem we're trying to solve?
-
-If a project supports `binstall` you can then install binaries via `cargo binstall NAME` where `NAME` is the name of the crate. We hope the defaults will work without configuration in _some_ cases, however, different projects have wildly different configurations. You may need to add some cargo metadata to support `binstall` in your project, see [Usage](#Usage) for details.
-
-
 ## Features
 
 - Manifest discovery
-  - [x] Fetch manifest from crates.io
-  - [ ] Fetch manifest using git
+  - [x] Fetch manifest via crates.io
+  - [ ] Fetch manifest via git
   - [x] Use local manifest (`--manifest-path`)
 - Package formats
   - [x] Tgz
   - [x] Tar
   - [x] Bin
+- Extraction / Transformation
+  - [ ] Extract from subdirectory in archive (ie. support archives with platform or target subdirectories)
+  - [ ] Extract specific files from archive (ie. support single archive with multiple platform binaries)
 - Security
   - [ ] Package signing
   - [ ] Package verification
 
-## Usage
+## Installing Binaries
+
+First you'll need to install `cargo-binstall` either via `cargo install cargo-binstall` (and it'll have to compile, sorry...), or by grabbing a pre-compiled version from the [releases](https://github.com/ryankurte/cargo-binstall/releases) page and putting that somewhere on your path. It's like there's a problem we're trying to solve?
+
+Once a project supports `binstall` you can then install binaries via `cargo binstall NAME` where `NAME` is the name of the crate. This will then fetch the metadata for the provided crate, lookup the associated binary file, and download this onto your system. 
+By default the latest version is installed, which can be overridden using the `--version` argument, and packages are installed to `$HOME/.cargo/bin` as is consistent with `cargo install`, which can be overridden via the `--install-path` argument. As always `--help` will show available options.
```

Comment:
Could be worth mentioning how you track/manage multi-versions, updates etc.

Reply:
hmm i'm not sure how to describe this rn :-/

Reply:
So .... you do allow mulitple versions to be installed? Or not?

Reply:
it'll clobber anything with the same name atm, extract is just a blind `.tgz` into output dir...

could specify binary names in the metadata then symlink against em instead which would be dope for atomic updating (and then this would also be an in-app atomic update library lol)

Reply:
Yeah, it probably is. I think homebrew does something similar. Allows fast rollbacks etc.

Something to keep an eye on is tar extraction attacks (../ or symlinks etc).

Reply:
> Something to keep an eye on is tar extraction attacks (../ or symlinks etc).

yeah the official policy on this is -yolo- at this time, mayybe a confirmation in the default case would be good to eh